packages/nx/src/generators/utils/project-configuration.spec.ts

```diff
@@ -339,6 +339,15 @@ describe('project configuration', () => {
         expect(configurations.get('test')).toEqual(baseTestProjectConfigV2);
         expect(configurations.get('test2')).toEqual(baseTestProjectConfigV2);
       });
+
+      it('should have JSON $schema in project configuration for standalone projects', () => {
```

Comment:
Can we add a test that uses ajv to ensure generated file matches the schema.json's to ensure they don't drift apart? This could be in e2e as well, but I think it would be faster as a unit test and easy enough.

Reply:
Done. It is a simple AJV test but we can expand later